backend/app/exporters/serializers/mods.rb

```diff
@@ -35,7 +35,7 @@ def serialize_mods_inner(mods, xml)
 
 
     xml.language {
-      xml.languageTerm(:type => 'code') {
```

Comment:
@avatar382 @cdibella Do we need to have both the language code and language term exported in two different XML tags? As a reference, I am looking at lines 19-27 in the spreadsheet attached to ANW-451.

Reply:
Yes, the intent is to have two different XML tags - one with a type "text" and the other with a type "code"